src/axom/inlet/Container.hpp

```diff
@@ -1231,6 +1231,34 @@ class Container : public Verifiable<Container>
   template <typename Func>
   void forEachCollectionElement(Func&& func) const;
 
+  /*!
+   *****************************************************************************
+   * \brief Applies a provided function to nested elements of the calling table
+   * and stores the result in a range
+   * 
+   * \pre The function must accept two arguments of type Table& and
```

Comment:
```suggestion
   * \pre The function \a func must accept two arguments of type Table& and
```

Reply:
Also, can you please check that doxygen doesn't choke on the ampersand in `Table&`

Reply:
I think Doxygen doesn't render private methods so this would only be for developer use